parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -42,9 +42,11 @@ done
 
 iptables-save >/etc/sysconfig/iptables
 
-sed -i -e "s#--master=.*#--master=https://$(hostname --fqdn):8443#" /etc/sysconfig/${SERVICE_TYPE}-master-api
+#sed -i -e "s#--master=.*#--master=https://$(hostname --fqdn):8443#" /etc/sysconfig/${SERVICE_TYPE}-master-api
```

Comment:
delete, don't comment

Reply:
I left this here for discussion.  In 3.9 this was putting the fqdn in the environment file for the API.  The equivalent now would be adding it to the `/etc/origin/master/master.env` file.  Is this still required?  I would think we still want this.